### DISCUSSION TITLE
Hardware Requirements

### DISCUSSION THREAD

**Discussion by thehappydinoa** [lucidrains/DALLE2-pytorch#10] [Q&A]
Hi there,

Love this work, super appreciative of a PyTorch port. I was wondering what the intended/recommended hardware specs are for running this model. 

Thanks,
Aidan

**Comment by lucidrains** [comment]
@thehappydinoa Hi Aidan! Thanks for the kind words

No one really knows for sure, as OpenAI is usually tight-lipped, but in one interview they revealed the model size was smaller than DALLE1, which is good news. It means that the hardware requirements should be no worse than https://github.com/ai-forever/ru-dalle

**Reply by LifeIsStrange** [reply]
see also:
https://github.com/openai/dalle-2-preview/issues/6

Also I believe we can do much better than openAI by leveraging the state of the art ZERO3 https://github.com/microsoft/DeepSpeed
